AgentDiscoveries-Backend/src/main/resources/config.properties

```diff
@@ -1,8 +1,8 @@
 
 # Database properties
 database.url = jdbc:mysql://localhost:3306/agentdiscoveries?useSSL=false
-database.username = CIA
-database.password = Amex1234
+database.username = agentdiscoveries

```

Comment:
Can we remove these files? Don't think we wanted this (or the persistence.xml one) committed?